Azure_Pipelines/azure-pipelines.normalization.cd.yaml

```
@@ -52,6 +52,13 @@ steps:
         echo "GIT_COMMIT: ${GIT_COMMIT}"
         echo "##vso[task.setvariable variable=GIT_COMMIT]${GIT_COMMIT}"
   
+  - task: PowerShell@2
+    displayName: "AppSettings Injection"
+    inputs:
+      targetType: 'filePath'
+      filePath: '$(Build.SourcesDirectory)/Scripts/git_commit.ps1'
+      pwsh: true
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)